### ISSUE TITLE
insert warning

### ISSUE THREAD

**Issue by bAndie91** [afilipovich/gglsbl#11]
I get many of the following warn message while downloading the database:
```
WARNING Failed to insert chunk because of columns value, chunk_number, list_name, chunk_type are not unique
```
Does it mean my database is inconsistent?

**Comment by afilipovich** [comment]
Several people reported similar issue and I observed it myself. It looks like the root cause is server sending data for already existing chunks. If these warning messages are annoying, deleting sqlite cache and re-syncing data should fix it.

I am closing the issue, please feel free to reopen if you have further info on that.

**Comment by afilipovich** [comment]
Reopening the issue as I found out that the root cause lies within the client code: by default SQLite does not enforce foreign keys, so hash_prefix entries do not get deleted by cascade when referenced chunk rows are deleted.

Working on a fix.

**Comment by afilipovich** [comment]
Fixed in v1.0.0.
